src/app/view/admin/admin-alldevices/admin-alldevices.component.html

```diff
@@ -228,7 +228,10 @@ <h6>Commissioning Date:</h6>
         <hr />
       </div>
       <div *ngIf="!hideMap">
-        <app-map #mapComponent></app-map>
+        <app-map
+          #mapComponent
+          (markerClicked)="onMarkerClick($event)"
```

Comment:
add condition button that will be shown only when the map filter has been selected that will have a black background color it will have the text "Reset Map Filter"

- Shows only when I'm filtering by map 
- Hides the moment I click on it